src/screens/Home/Home.js

```diff
@@ -58,44 +88,111 @@ export function Home({ navigation }) {
   }
 
   const Movie = ({ item }) => (
-    <View>
+    <View style={styles.container_items_movies}>
       <TouchableHighlight accessibilityRole="button" onPress={() => goDetails(item)}>
-        <View style={styles.item} accessibilityIgnoresInvertColors={true}>
-          <View style={styles.avatarContainer}>
-            <Image style={styles.avatar} source={{ uri: `${IMAGE_URL + item.backdropPath}` }} />
-          </View>
-          <Text style={styles.name}>{item.originalTitle}</Text>
-        </View>
+        <Image
+          style={styles.avatar}
+          accessibilityIgnoresInvertColors={true}
+          source={{ uri: `${IMAGE_URL + item.backdropPath}` }}
+        />
       </TouchableHighlight>
       <TouchableWithoutFeedback
         accessibilityRole="button"
-        onPress={() => addMovieFavourites(item)}
+        onPress={() => addMovieFavorites(item)}
         accessibilityIgnoresInvertColors={true}
       >
-        <Image source={addIcon} />
+        <Image style={styles.icon_add} source={addFavoriteMovieIcon} />
       </TouchableWithoutFeedback>
     </View>
   );
 
-  const headerComponent = () => <Text style={styles.listHeadLine}>{MOVIES}</Text>;
+  const MovieFavorites = ({ item }) => (
+    <View style={styles.container_items_movies}>
+      <TouchableHighlight accessibilityRole="button" onPress={() => goDetails(item)}>
+        <Image
+          style={styles.avatar}
+          accessibilityIgnoresInvertColors={true}
+          source={{ uri: `${IMAGE_URL + item.backdropPath}` }}
+        />
+      </TouchableHighlight>
+      <TouchableWithoutFeedback
+        accessibilityRole="button"
+        onPress={() => removeMovieFavs(item)}
+        accessibilityIgnoresInvertColors={true}
+      >
+        <Image style={styles.icon_add} source={addFavoriteMovieIcon} />
+      </TouchableWithoutFeedback>
+    </View>
+  );
 
   const emptyComponent = () => <Text>{EMPTY_MOVIES}</Text>;
-
-  const itemSeparator = () => <View style={styles.separator} />;
+  const removeMovieFavs = (item) => {
+    dispatch(deleteMovie(item));
+  };
 
   if (
     isConnected === true ||
     (isConnected !== true && data != null && isOneDayDiff(date, new Date()))
   ) {
     return (
-      <FlatList
-        ListHeaderComponentStyle={styles.listHeader}
-        ListHeaderComponent={headerComponent}
-        data={data.data.results}
-        renderItem={Movie}
-        ItemSeparatorComponent={itemSeparator}
-        ListEmptyComponent={emptyComponent}
-      />
+      <SafeAreaView style={styles.safe_area}>
+        <ScrollView style={styles.container}>
+          <ImageBackground
+            source={require(image_background)}
+            style={styles.image}
+            resizeMode="cover"
+          >
+            <LinearGradient colors={['#00000000', '#000000']} style={styles.gradientHeader}>
```

Comment:
These gradient colors should be placed elsewhere (maybe a `constants/gradients.js` file).

Reply:
Done!